apps/storefront/src/shared/service/request/base.ts

```diff
@@ -16,18 +16,19 @@ const ENVIRONMENT_B2B_APP_CLIENT_ID: EnvSpecificConfig<string> = {
 };
 // cspell:enable
 
-function getAPIBaseURL() {
-  const environment: Environment = window.B3?.setting?.environment ?? Environment.Local;
+const DEFAULT_ENVIRONMENT =
+  import.meta.env.VITE_DEFAULT_TO_LOCAL === 'TRUE' ? Environment.Local : Environment.Production;
```

Comment:
default to the production environment so that builds in ci/cd don't have to change.